cluster/images/conformance/Dockerfile

```diff
@@ -14,12 +14,21 @@
 
 ARG BASEIMAGE
 
+FROM gcr.io/k8s-staging-build-image/debian-base:v2.1.3 as base
+RUN mkdir /tmp/results \
+ && chown 65532 /tmp/results # Distroless nonroot UID
+
 FROM ${BASEIMAGE}
 
+COPY --from=base /tmp/results /tmp
+
+# This is a dependency for `kubectl diff` tests
+COPY --from=base /usr/bin/diff /usr/local/bin/
+
 COPY ginkgo /usr/local/bin/
 COPY e2e.test /usr/local/bin/
 COPY kubectl /usr/local/bin/
-COPY run_e2e.sh /run_e2e.sh
+COPY gorunner /run_e2e.sh
```

Comment:
@wilsonehusin is that COPY correct ?

Reply:
@vladimirvivien Yeah this is to preserve folks invoking `/run_e2e.sh` so that it won't immediately break their automation. I'm working on getting a deprecation message printed on that, (4) above might be relevant -- if we set `ENTRYPOINT` for this image then we don't need to worry about this abstraction anymore.

Reply:
This is still a breaking change either way (no shell utils available, script can't be sourced, etc.). Though I'm not sure if any users depended on that.

Reply:
we should consider dropping this copy in the future, since we're storing two copies of the binary ..